desktop.css

```diff
@@ -0,0 +1,629 @@
+  justify-content: center;
+  align-items: center;
+  padding-bottom: 400px;
+  height: fit-content;
+  padding-top: 50px;
+  width: 100vw;
+  left: 0;
+  margin-left: -5px;
+  overflow-x: hidden;
+}
+
+.About ul li {
+  color: #fff;
+  width: 164px;
+  height: 24px;
+  font-family: 'Crete Round', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  margin: 0 auto;
+  text-align: justify;
+}
+
+.head {
+  font-family: 'Inter', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 35px;
+  line-height: 20px;
+}
+
+.list {
+  row-gap: 20px;
+  display: grid;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-content: center;
+  margin-top: -65px;
+  text-align: justify;
+  width: fit-content;
+}
+
+.get-resume {
+  position: relative;
+  width: 180px;
+  padding: 7px;
+  height: fit-content;
+  background: #ff6b00;
+  display: flex;
+  justify-content: center;
+  font-weight: 500;
+  color: #fff;
+  font-family: 'Inter', Arial, Helvetica, sans-serif;
+  align-self: flex-start;
+  margin-left: 265px;
+  margin-top: 35px;
+}
+
+.image-2 {
+  height: 400px;
+  width: 50%;
+  background-image: url(./img/desktop/laay.png);
+  background-repeat: no-repeat;
+  position: absolute;
+  top: 7%;
+  left: 59%;
+}
+
+.heading > ul {
+  list-style: none;
+  align-items: center;
+}
+
+.heading > ul > li {
+  font-size: 5px;
+  vertical-align: middle;
+  text-align: center;
+  padding: 5px 0;
+}
+
+h3 {
+  text-align: center;
+  font-size: 32px;
+  font-weight: 600px;
+  color: #fff;
+}
+
+.empty {
+  height: 10px;
+}
+
+.descripton {
+  width: 505px;
+  height: 148px;
+  font-family: 'Inter', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 24px;
+  color: #000;
+  flex: none;
+  flex-grow: 0;
+}
+
+.description-abt {
+  margin-left: -78%;
+}
+
+.rule-2 {
+  position: absolute;
+  width: 70%;
+  top: 538px;
+}
+
+.description-2 {
+  margin: auto;
+  text-align: left;
+  font-family: 'Inter', Arial, Helvetica, sans-serif;
+  font-size: 40px;
+  height: 180px;
+  width: 437px;
+  line-height: 52px;
+  color: #172b4d;
+  position: relative;
+  margin-top: 3px;
+}
+
+.section_3 {
+  width: 100vw;
+  display: flex;
+  flex-direction: row;
+  margin-left: -6%;
+}
+
+#name-input,
+#email-input {
+  font-family: 'Inter', Arial, Helvetica, sans-serif;
+  width: 600px;
+  border: 0;
+  border-bottom: 1px solid gray;
+  outline: 0;
+  font-size: 10px;
+  color: rgb(14, 12, 12);
+  padding: 7px 0;
+  background: transparent;
+  transition: border-color 0.2s;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  row-gap: 10px;
+}
+
+#more-text {
+  border: 0;
+  color: #000;
+  background: hsl(0, 11%, 95%);
+  font-style: normal;
+  font-family: 'Inter', Arial, Helvetica, sans-serif;
+  margin-left: -73%;
+}
+
```

Comment:
Try to make a design as close to FIGMA design as possible.

 Your Design:
 ![image](https://user-images.githubusercontent.com/61361037/213814931-35349048-b09f-4a53-903c-bfaaa751b318.png)
FIGMA Design:
![image](https://user-images.githubusercontent.com/61361037/213815157-ff69d8a0-cf9e-4f3f-9c3a-699bdd15ebd3.png)